vk_video_decoder/libs/NvVideoParser/include/cpudetect.h

```diff
@@ -18,11 +18,16 @@ static int inline count_trailing_zeros(unsigned long long resmask)
     const unsigned char dummyIsNonZero =_BitScanForward64(&offset, resmask); // resmask can't be 0 in this if
 #elif _WIN32
     unsigned long offset = 0;
-    const unsigned char isNonZero = _BitScanForward(&offset, (unsigned long)(resmask >> 32U)); // resmask can't be 0 in this if
-    if (isNonZero) {
-        offset += 32U;
-    } else {
-        _BitScanForward(&offset, (unsigned long)resmask); // resmask can't be 0 in this if
+    unsigned long resmaskLsb = (unsigned long)(resmask & 0xFFFFFFFFULL);
+    if (resmaskLsb != 0U) {
+        const unsigned char dummyIsNonZero = _BitScanForward(&offset, resmaskLsb); // resmaskLsb can't be 0 in this if
+    }
+    else {
+        unsigned long resmaskMsb = (unsigned long)(resmask >> 32U);
+        const unsigned char isNonZero = _BitScanForward(&offset, resmaskMsb); // resmaskMsb can't be 0 in this if
+        if (isNonZero) {
```

Comment:
I dont think this branch is necessary as the comment above says that  `resMask` cant be zero and so `resmaskMsb` will be different from 0 so _BitScanForward will return something different from zero IIUC